.gitlab-ci.yml

```diff
@@ -863,6 +863,7 @@ changelog:
     - test -z "$RELEASE_PLEASE_PR" && echo "No release-please PR found" && exit 0
     - echo "INFO - generating changelog notes from git cliff"
         && gh pr checkout $RELEASE_PLEASE_PR
+        && git pull
```

Comment:
Couldn't this potentially cause merge conflicts?
What about a simple `git fetch github-${TS_SUFFIX} pull/$RELEASE_PLEASE_PR/head && git reset --hard FETCH_HEAD`?

Reply:
yeah, thanks for the trick!